docs/faq.rst

```diff
@@ -0,0 +1,102 @@
+.. _faq:
+
+Frequently asked questions
+==========================
+
+General fairness questions
+--------------------------
+
+Where can I learn more about fairness in machine learning?
+    In our :ref:`user guide <user_guide>` we provide links to the papers describing
+    Fairlearn's algorithms.
```

Comment:
maybe replace with -> Fairlearn's unfairness mitigation algorithms.

Not necessarily the first topic I would point people towards, though, because to mitigate you first need to understand why it's complex, what the sociotechnical context of your application scenario is, etc.
I like that you're pointing people to "further reading" and the user guide for Fairness in ML, but perhaps I would recommend removing parts of the first sentence. What do you think?

Reply:
Good point. I've switched the sentences around, and rephrased them a bit.